neo/rawio/read_tif.py

```diff
@@ -0,0 +1,18 @@
+from neo.core import ImageSequence
```

Comment:
I've used this function as a basis to make a full IO module to go in `neo/io/tiffio.py`:
```python
 # -*- coding: utf-8 -*-
"""
Neo IO module for optical imaging data stored as a folder of TIFF images.

"""

import os
from PIL import Image
import numpy as np
from neo.core import ImageSequence, Segment, Block
from .baseio import BaseIO


class TiffIO(BaseIO):
    """
    Neo IO module for optical imaging data stored as a folder of TIFF images.
    """
    name = 'TIFF IO'
    description = "Neo IO module for optical imaging data stored as a folder of TIFF images."

    _prefered_signal_group_mode = 'group-by-same-units'
    is_readable = True
    is_writable = False

    supported_objects = [Block, Segment, ImageSequence]
    readable_objects = supported_objects
    writeable_objects = []

    support_lazy = False

    read_params = {}
    write_params = {}

    extensions = []

    mode = 'dir'

    def __init__(self, directory_path=None, **kwargs):
        BaseIO.__init__(self, directory_path, **kwargs)

    def read(self, lazy=False, units=None, sampling_rate=None, spatial_scale=None, **kwargs):
        if lazy:
            raise ValueError('This IO module does not support lazy loading')
        return [self.read_block(lazy=lazy, units=units, sampling_rate=sampling_rate,
                                spatial_scale=spatial_scale, **kwargs)]

    def read_block(self, lazy=False, units=None, sampling_rate=None,
                   spatial_scale=None, **kwargs):

        # find all the images in the given directory
        # TODO: only load files with the extension .tif or .tiff (see the glob module)
        file_name_list = os.listdir(self.filename)
        list_data_image = []
        for file_name in file_name_list:
            list_data_image.append(np.array(Image.open(self.filename + "/" + file_name), dtype=np.float))
        # todo: find out if this works for colour TIFFs, not just monochrome
        print("read block")
        image_sequence = ImageSequence(np.stack(list_data_image),
                                       units=units,
                                       sampling_rate=sampling_rate,
                                       spatial_scale=spatial_scale)
        print("creating segment")
        segment = Segment(file_origin=self.filename)
        segment.annotate(tiff_file_names=file_name_list)
        segment.imagesequences = [image_sequence]

        block = Block(file_origin=self.filename)
        segment.block = block
        block.segments.append(segment)
        print("returning block")
        return block
```

Could you do the same for `read_blk.py` --> `neo/io/blkio.py` and `read_txt.py` --> `neo/io/asciiimageio.py`?